Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/RootAssetBrowserEntry.cpp

```diff
@@ -391,6 +393,40 @@ namespace AzToolsFramework
                 return parent;
             }
 
+            // If the project is not inside the root engine folder
+            if (!m_projectPath.IsRelativeTo(m_enginePath))
+            {
+                // Update the parent to be the project directory
+                AZ::IO::FixedMaxPath directory;
+                if (absolutePathView.IsRelativeTo(m_projectPath) && !inProjectDirectory)
+                {
+                    for (auto pathIter = m_projectPath.begin(); pathIter != m_projectPath.end(); ++pathIter)
+                    {
+                        if (std::next(pathIter) == m_projectPath.end())
+                        {
+                            break;
+                        }
+                        directory = directory / *pathIter;
```

Comment:
This logic is just copying the `m_projectPath` to the `directory`
```suggestion
                directory = m_projectPath;
```

Reply:
`m_projectPath` returns the path with the project folder, so this logic copies the path leading up to the project folder, which is used as the parent path. I wasn't sure of a better method to get the path without the project folder at the end.

Reply:
There is a [ParentPath](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/IO/Path/Path.h#L202-L206) function that should be used

Reply:
Oh awesome, thank you I'll update it 👍